configs/maheswaranathan_2023_LNP.yaml

```
@@ -19,7 +19,7 @@ seed: 42
 check_stimuli_responses_match: false
 
 data:
-  root_dir: ???
+  root_dir: ${oc.env:OPENRETINA_CACHE_DIRECTORY}
   data_dir: ${data.root_dir}/baccus_data/neural_code_data/ganglion_cell_data/
   output_dir: ${data.root_dir}/openretina_assets/runs
```

```suggestion
  ${hydra:runtime.output_dir}
```
for consistency with the other files.